### DISCUSSION TITLE
port forwarding isn't work

### DISCUSSION THREAD

**Discussion by nguyenduchuyz190** [community/community#102570] [Codespaces]
### Select Topic Area

Question

### Body

- VS Code Version: 1.85.2
- OS Version: MacOS 14
- Codepsace Version: 1.16.9

Steps to Reproduce:

1. Create a codespace from empty github
2. Create an vite project
3. Start dev server

<img width="1131" alt="Screenshot 2024-01-29 at 05 40 57" src="https://github.com/microsoft/vscode/assets/124940988/c64b2bd1-4d57-4843-a079-9e5a96cf336e">
<img width="864" alt="Screenshot 2024-01-29 at 05 40 37" src="https://github.com/microsoft/vscode/assets/124940988/36fff0d7-73b5-4055-92f2-71a70fd415f5">
<img width="745" alt="Screenshot 2024-01-29 at 05 43 36" src="https://github.com/microsoft/vscode/assets/124940988/5ea48c06-8781-429a-b3f2-be5385981de3">

**Comment by ghost** [comment]
If this is codespace maybe try making a public port because unless you have actual software installed like your discussio says your on codespaces try making a public port and click the ebutton that forwrds you to the port automatically try switching to the codespace  if your connecting codespace to vscode

**Reply by ghost** [reply]
I am not sure what  your trying to tell me
